detoks/src/main/java/nl/tudelft/trustchain/detoks/TorrentManager.kt

```diff
@@ -240,6 +243,7 @@ class TorrentManager private constructor (
     }
 
     fun addTorrent(hash: Sha1Hash, magnet: String) {
+        profile.incrementTimesSeen(MagnetLink.hashFromMagnet(magnet))
```

Comment:
Maybe check first if there was an mp4 in it, otherwise it won't be added and shouldn't be tracked